docs/medical-api/api-reference/patient/delete-patient.mdx

```diff
@@ -27,8 +27,8 @@ import { MetriportMedicalApi } from "@metriport/api-sdk";
 
 const metriport = new MetriportMedicalApi("YOUR_API_KEY");
 
-const patientId = "2.16.840.1.113883.3.666.5.2004.2.110";
-const facilityId = "2.16.840.1.113883.3.666.5.2004.4.102";
+const patientId = "018a80c4-292a-7486-a1234-76yuhe23yu14";
+const facilityId = "018a80c4-292a-7486-a1234-9uiu76yhe234";
```

Comment:
double checking that these are fake UUIDs, and not from prod?

Reply:
Yes -- I made them up 👍